### DISCUSSION TITLE
xDrip stopped working after popup about calibration

### DISCUSSION THREAD

**Discussion by Jarlsten** [NightscoutFoundation/xDrip#2487] [Q&A]
Hi,

This morning I got a popup from xDrip about some calibration thing ( I did not read that carefully :-( ).
And now does not xDrip read my sensor.
I have tried the normal stuff when the reading stopps such as disconnect the miaomiao and connect it again.
I have a Miaomiao version1 and libre sensor version1.

Does any one know how to set back the update or what I need to do?

Mvh
Håkan

**Comment by AliQ101** [comment]
Hi,
are Bluetooth Device: connected or only connecting and also Phone Service State: connected on Status page/BT collector status page?

**Reply by Jarlsten** [reply]
![Screenshot_20221121_182020_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/4248930/203122265-6b83cc93-5b28-441c-bfa1-3418050cf20f.jpg)

**Reply by Jarlsten** [reply]
![Screenshot_20221121_183611_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/4248930/203122990-40afcfa5-ac5e-4f84-842d-82e620623079.jpg)

**Reply by Navid200** [reply]
Have you disabled battery optimization for xDrip in Android settings on your phone?

**Reply by Jarlsten** [reply]
Yes I have

**Comment by Jarlsten** [comment]
Thanks for helping.
Is this what you were asking for?

**Reply by Navid200** [reply]
Slowest wakeup, 4 minutes late means that xDrip was not able to wake up when it expected it would wake up.  This usually happens because the phone disables apps in the background.
One of the ways to remedy this on some phones is to disable battery optimization for xDrip in Android settings.
I don't know if you have done that or not.

There are also phones that are known to have problems like Xiaomi:
https://dontkillmyapp.com/

**Reply by Jarlsten** [reply]
XDrip have worked fine for a long time. Then this morning after this "calibration "update it stopped. I have not changed any settings. Do you think the update changed this?

**Reply by Navid200** [reply]
The battery optimization setting could change after an Android patch.
But, verifying that setting is very easy and it would be easier to check it than to have a debate whether it may have changed or not.

**Reply by Jarlsten** [reply]
![Screenshot_20221121_191152_com huawei systemmanager](https://user-images.githubusercontent.com/4248930/203129505-648c4082-7f16-4860-b1cf-eb5d5da94b0d.jpg)

**Reply by Jarlsten** [reply]
Is this right?

**Reply by Jarlsten** [reply]
Sorry for the Swedish 😀

**Reply by Navid200** [reply]
Those are all good.  However, please go to Android settings.  Find and select xDrip.
Now, tap on Advanced.
Now, tap on Battery.
Now, you can see if battery optimization is active or not.

**Reply by Jarlsten** [reply]
![Screenshot_20221121_192521_com android settings](https://user-images.githubusercontent.com/4248930/203131793-9ec6d720-b3cd-4e5d-a005-e0bdd3625d52.jpg)

**Comment by AliQ101** [comment]
You said that there was some calibration pop-up message. I would try to delete all calibrations previously made that are still active for this sensor and make new one calibration (just based on FSL reader, not blood) to see if it works. Sometimes might help and it worked for me. Also MM battery seems low (for me always stopped around 60%) but MM BT connection seems ok.

**Reply by Jarlsten** [reply]
Yes I got a popup that said something about "automated calibration" and I pressed Yes. :-( 
I have deleted all calibrations previously made. Still does not work. :-(
I have stopped the sensor and started it again. Still no vallues. :-(

**Comment by Jarlsten** [comment]
Thanks for all help and time you have spent on my case.
This was my last libre1 sensor so started my first Libre 2 sensor and this seem to be working fine.

**Comment by Navid200** [comment]
@psonnera

**Reply by Navid200** [reply]
Sorry, I missed the message that it is now working with Libre 2.  I would not have tagged you if I had seen that.  Sorry

**Reply by Jarlsten** [reply]
No worries. Thanks for taking the time.
